### ISSUE TITLE
No Apple Device in DFU mode

### ISSUE THREAD

**Issue by aparajey** [MatthewPierson/1033-OTA-Downgrader#79]
Hello,

i am using MacOS Catalina and when i am running restore.sh i am getting following error. I am in DFU mode but script is not detecting Iphone.

2019-11-19 02:25:24.662 system_profiler[50127:1074590] SPUSBDevice: IOCreatePlugInInterfaceForService failed 0xe00002be
2019-11-19 02:25:24.671 system_profiler[50127:1074590] SPUSBDevice: IOCreatePlugInInterfaceForService failed 0xe00002be
No matching processes belonging to you were found
Waiting 10 seconds to allow you to enter DFU mode
Attempting to get into pwndfu mode
Please just enter DFU mode again on each reboot
The script will run ipwndfu again and again until the device is in PWNDFU mode
ERROR: No Apple device in DFU Mode 0x1227 detected after 5.00 second timeout. Exiting.

Any suggestion?

**Comment by MatthewPierson** [comment]
Try a different cable/USB port. Does iTunes detect the device in DFU mode?

**Comment by aparajey** [comment]
yes iTunes detect Iphone in recovery/DFU mode. but script doesn't

**Comment by qwqcode** [comment]
i have the same issue.
TAT

**Comment by qwqcode** [comment]
I downgraded to mojave and it works well...

**Comment by cs0663c** [comment]
Attempting to get into pwndfu mode
Please just enter DFU mode again on each reboot
The script will run ipwndfu again and again until the device is in PWNDFU mode
ERROR: No Apple device in DFU Mode 0x1227 detected after 5.00 second timeout. Exiting.
No matching processes belonging to you were found
Waiting 10 seconds to allow you to enter DFU mode

**Comment by jinjin123** [comment]
> Attempting to get into pwndfu mode
> Please just enter DFU mode again on each reboot
> The script will run ipwndfu again and again until the device is in PWNDFU mode
> ERROR: No Apple device in DFU Mode 0x1227 detected after 5.00 second timeout. Exiting.
> No matching processes belonging to you were found
> Waiting 10 seconds to allow you to enter DFU mode

same problem,I just downgraded my macpro2015 from mojave to high sierra,cannot test mojava ....

**Comment by Stavi1010** [comment]
I am currently having the same issue but my iTunes is not detecting the device in dfu mode. What do I do?

**Comment by lakiyo** [comment]
I have the same problem. macOS Catalina 10.15.6, iPad mini 2 

just infinity loop

**Comment by zhangguorong1995** [comment]
I have the same problem. macOS Catalina 10.15.6, iPhone 5s

just infinity loop
